.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 *.js
 *.map
 *.tgz
+.DS_Store
```

Comment:
We don't include this in the project because this should be configured in user space.